.github/CODEOWNERS

```diff
@@ -5,4 +5,4 @@
 # The last matching pattern has the most precendence.
 
 # These *code owners* will be the default owners for everything in the repository.
-* @alkwa-msft @jamesburnside @anjulgarg @PorterNan @mgamis-msft @alcail @prprabhu-ms @dmceachernmsft
\ No newline at end of file
+* @alkwa-msft @jamesburnside @anjulgarg @PorterNan @mgamis-msft @alcail @prprabhu-ms @dmceachernmsft @carocao-msft @edwardlee-msft @prabhjot-msft
```

Comment:
remove alcail also (unless she wants to review lots of our PRs still 😁)

Reply:
Sure. Removed. She's welcome to help us review but I'm sure she doesn't want the extra emails :).